learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -0,0 +1,162 @@
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- A command terminalA text editor.
+- [VSCode](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed is our preferred choice, and it has good support for Ballerina.
+- A Ballerina installation. Follow the steps in [Installing Ballerina](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/).
+- Some experience with at least one programming language. 
+
+## Meet `bal`
+
+`bal` is the Ballerina build tool and package manager. Among other things, `bal` helps you to create, build, test, and run your project. 
+
+You can get the latest `bal` tool version with the Ballerina installation. Open your terminal and run the following commands just to make sure everything is ready:
+
+```
+$bal
+$bal version
+```
+
+## Create a new project
+
+Let’s write a Ballerina program that prints “Hello, World!”. Use the `bal new` command to create a new Ballerina project. 
+
+```
+$ bal new greeter
+```
+
+This command creates a new directory called `greeter`. Let’s move into this directory to see what’s in there. 
+
+```
+greeter/
+├── Ballerina.toml
+└── main.bal
+```
+
+- `Ballerina.toml` contains metadata that describes your project. Also, `bal` tool uses `Ballerina.toml` to identify the root of a project.
+- `main.bal` is a source file, and it should contain Ballerina code that prints “Hello, World!” to the console. You can addput any number of source files into the `greeter` directory.
+
+## Say “Hello, World!”
+
+You can open the project directory in your text editor. If you are using VSCode, run `code .` inside the `greeter` directory. Then, open `main.bal` to see the generated source.
+
+```ballerina
+import ballerina/io;
+
+public function main() {
+    io:println("Hello, World!");
+}
+```
+
+In this code:
+
+- The first line is an import statement that makes the capabilities in the `[ballerina/io](https://lib.ballerina.io/ballerina/io/latest)` module available to your program. This module contains functions to write to the console, read from the console, and perform read/write operations on files.,
+- The `main` function is your program’s entry point, and you can execute it by running the program. 
+- This function contains a statement that prints `Hello, World!` to the console. This statement calls the `println` function in the `io` with `Hello, World!`.
+
+If you are interested in learning more about the language, read our [Language Basics](https://ballerina.io/learn/language-basics/) guide. 
+
+Let’s run this project by running `bal run` in your terminal:
+
+```
+$ bal run
+Compiling source
+	example/greeter:0.1.0
+
+Running executable
+
+Hello, World!
+```
+
+You can also generate an executable program with `bal build`:
+
+```$ bal buildCompiling source	sameera/greeter:0.1.0
+Generating executable	target/bin/greeter.jar
+```
+
+Then, use `bal run` as follows.
```

Comment:
```suggestion
Then, use `bal run` as follows.
```
Shall we add the instructions as numbered bullet points as they seem sequential?

Reply:
Will check this in the next review.